android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java

```diff
@@ -249,6 +277,35 @@ public void onAnimationEnd(Animator animation) {
                     }
                 }
             );
+
+        audioControls
+            .animate()
+            .alpha(0f)
```

Comment:
I would add a bit of alpha, it looks nice. But ask Moritz

Reply:
I think the current animation looks smooth enough.